project/project.go

```diff
@@ -337,6 +383,16 @@ func (p *Project) Down(ctx context.Context, opts options.Down, services ...strin
 		return err
 	}
 
+	if opts.RemoveVolume {
```

Comment:
Should this be an option for `Delete` as well? I think the `rm` command also has a `-v` flag.